.github/pull_request_template.md

```diff
@@ -9,15 +9,3 @@
 ## Documentation <sub><sup>(that requires updating in Notion / Github)</sup></sub>
 
 ## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>
-
-## Checklist
-- [ ] Add a brief description, screenshots (if applicable), and testing instructions for this PR.
-- [ ] Introduce or modify relevant documentation
-- [ ] Ensure unhappy paths are considered/tested, and there is a plan for monitoring post-deployment
-- [ ] Test your changes in the deployed PR preview environment / [run migration locally](https://github.com/clearbanc/code/tree/staging/packages/migrations#run-migrations) (if applicable).
-- [ ] Ensure test coverage value in config file match actual coverage (if applicable).
```

Comment:
I'm starting to realize that the PR checklist is more for the reviewer than for the person submitting the PR. Any thoughts on having a separate checklist where we can provide more guidance on reviewing PRs (we can tackle this separately).